### ISSUE TITLE
jdbc: Support read only connection

### ISSUE THREAD

**Issue by nicktorwald** [tarantool/tarantool-java#176]
Implement both `Connection.isReadOnly` and  `Connection.setReadOnly` methods.
We need at least Tarantool DB support for that. It seems it should be implemented as part of transaction support how it has already done in Postgres ([docs](https://www.postgresql.org/docs/current/sql-set-transaction.html)).

For instance, Postgres driver executes `SET SESSION CHARACTERISTICS AS TRANSACTION READ ONLY` when a read-only flag is set.

**Comment by Totktonada** [comment]
It is the session parameter and I see no connection with transactions here. Anyway, we cannot implement it right now, so I filed the issue to tarantool and leave the milestone here as is for now. Let's return here when the issue in tarantool will be resolved.

Blocked by https://github.com/tarantool/tarantool/issues/4205
